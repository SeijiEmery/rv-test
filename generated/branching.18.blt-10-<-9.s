blt a0, a1, l1
    addi t1, zero, 0
l1:
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
