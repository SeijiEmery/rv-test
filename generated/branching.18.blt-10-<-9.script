load /x 0 generated/branching.18.blt-10-<-9.hex
writereg 10 10
writereg 11 9
writereg 6 1
setpc 0
run 1004
readreg 10
readreg 11
readreg 6