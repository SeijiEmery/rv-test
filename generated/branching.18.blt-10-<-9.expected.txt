tests/branching.test.s:190
R10 = 10
R11 = 9
R6 = 0
